src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/registries/routeRegistry.js

```diff
@@ -48,7 +34,7 @@ class RouteRegistry {
         });
     }
 
-    get(name: string): Route {
+    get(name: string): RouteType {
```

Comment:
Doesn't that return a `Route` now?

Reply:
Removed the interface so all which did use the `type Route` before use now the `class Route`